nearbridge/contracts/NearBridge.sol

```diff
@@ -200,117 +183,137 @@ contract NearBridge is INearBridge {
         require(votedFor > bpInfo.totalStake.mul(2).div(3), "NearBridge: Less than 2/3 voted by the block after next");
     }
 
-    function addLightClientBlock(bytes memory data) public payable {
+    struct BridgeState {
+        uint currentHeight;
+        uint nextTimestamp;
+        uint nextValidAt;
+        uint numBlockProducers;
+    }
+
+    function bridgeState() public view returns (BridgeState memory res) {
+        if (block.timestamp < lastValidAt) {
+            res.currentHeight = head.height;
+            res.nextTimestamp = untrustedHead.timestamp;
+            res.nextValidAt = lastValidAt;
+            res.numBlockProducers =
```

Comment:
It is unclear why do we not return `numBlockProducers` when there is an untrusted head.

Reply:
We do return the number of block producers when there is an untrusted head. This number is used by the watchdog to determine the number of signatures to check.